content-services/content-sources-backend
# Fixes 1968: Update RepositoryResponse with snapshot taskID

## Summary

Add a new parameter "last_snapshot_task_uuid" to the RepositoryResponse (repositoryList/Fetch/Edit responses)

When a repository enques a task to snapshot, the api will now return the above parameter.
<img width="475" alt="Screen Shot 2023-08-23 at 2 11 47 PM" src="https://github.com/content-services/content-sources-backend/assets/38083295/ad3ae1d4-16da-4bc2-b137-e83be78416d3">

This UUID can be confirmed by checking the taskInfo in the Admin tasks view under the ViewDetails Modal  > UUID (for the associated task of course).

<img width="416" alt="Screen Shot 2023-08-23 at 2 12 07 PM" src="https://github.com/content-services/content-sources-backend/assets/38083295/d4e71890-432e-48ea-b577-9a3f69f86b9c">

## Suggested Testing steps
- Create or Edit a repository with snapshotting enabled, check the response in the network for the "last_snapshot_task_uuid". 
- Copy the ID from the network tab above and check it against the  in the tasksInfo modal in the associated task.
